backend/app/config.py

```
@@ -0,0 +1,20 @@
+"""
+Configuration management using Pydantic settings.
+Loads environment variables with validation and type checking.
+"""
+from pydantic_settings import BaseSettings
+from typing import Optional
+
+class Settings(BaseSettings):
+    DATABASE_URL: str = "postgresql://portfolio:portfolio_dev_pass@postgres:5432/portfolio"
+    SECRET_KEY: str = "dev_secret_key_change_in_production"
```

Ruff (0.14.1)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)